Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -760,26 +554,6 @@ - (void)viewWillAppear:(BOOL)animated
     [self setBarButtonItemsForDisappearingMessagesConfiguration:configuration];
     [self setNavigationTitle];
 
-    // Other views might change these custom menu items, so we
-    // need to set them every time we enter this view.
-    SEL saveSelector = NSSelectorFromString(@"save:");
```

Comment:
I've heavily reworked how we use `UIMenuController` which should avoid these recurrent "sometimes wrong items appear" bugs.  We should always explicitly configure the menu controller items right before we  show it.  There's more work to be done in other places.